.circleci/config.yml

```diff
@@ -75,14 +75,27 @@ commands:
           destination: test_results
 
 jobs:
-  unit_test:
+  linting_checks:
     <<: *job_defaults
     steps:
       - test_setup
       - run:
-          name: Check Python lint
+          name: Check Python lint with yapf
           working_directory: ~/curation
           command: tools/yapf_lint.sh
+      - run:
+          name: Pylint on data_steward
+          working _directory: ~/curation
+          command: pylint -E data_steward/
```

Comment:
Could you please wrap this in a shell script similar to `tools/yapf_lint.sh`? It will allow us to more easily run the same exact command locally and also ensure it could be run from any subdirectory.

Reply:
We said we would defer wrapping pylint in a script for later